src/AzureAppConfigurationImpl.ts

```diff
@@ -258,29 +294,42 @@ export class AzureAppConfigurationImpl implements AzureAppConfiguration {
     }
 
     async #loadFeatureFlags() {
-        const featureFlagSettings: ConfigurationSetting[] = [];
-        for (const selector of this.#featureFlagSelectors) {
-            const listOptions: ListConfigurationSettingsOptions = {
-                keyFilter: `${featureFlagPrefix}${selector.keyFilter}`,
-                labelFilter: selector.labelFilter
-            };
+        // Temporary map to store feature flags, key is the key of the setting, value is the raw value of the setting
+        const funcToExecute = async (client) => {
+            const featureFlagSettings: ConfigurationSetting[] = [];
+            const selectors = JSON.parse(

```

Comment:
Any special reason to do serialize and deserialize the ff selectors?

Reply:
I'm not sure if it's for pageEtag? @Eskibear

Reply:
@linglingye I didn't find this code in the original implementation. What's the reason for this change?

The type of this.#featureFlagSelectors is `PagedSettingSelector[]` it is already an array. I didn't see any reason to stringfy it and then deserialize it.

Reply:
Discussed offline. This is for deep copy. There is no elegant way to do deep copy in javascript.